protocols/JabberG/src/jabber_opt.cpp

```diff
@@ -352,7 +352,6 @@ class CDlgOptAccount : public CJabberDlgBase
 	CCtrlButton		m_btnRegister;
 	CCtrlButton		m_btnUnregister;
 	CCtrlButton		m_btnChangePassword;
-	CCtrlHyperlink	m_lnkServers;

```

Comment:
Here you removed the variable declaration, but you still use it below.

Reply:
Fixed.